packages/math/init.lua

```diff
@@ -138,6 +143,10 @@ By default, this package uses Libertinus Math, so it will fail if Libertinus Mat
 Another font may be specified via the setting \autodoc:setting{math.font.family}.
 If required, you can set the font style and weight via \autodoc:setting{math.font.style} and \autodoc:setting{math.font.weight}.
 The font size can be set via \autodoc:setting{math.font.size}.
+The \autodoc:setting{math.font.features} setting can be used to specify OpenType features for the math font.
+It defaults to \code{+ssty} (script style), notably to ensure that some symbols such as the prime, double prime, etc. are displayed correctly.
+The default setting applies to Libertinus Math and well-designed math fonts, but some fonts may require different features.
+(The STIX Two Math font has a stylitic set \code{+ss04} from primes only, but also supports them with \code{+ssty}, which provides other optical adjustments.)
```

Comment:
Well we _could_ do it by enabling the feature based on isScriptMode / isScriptScriptMode rather than globally, that _should_ work in the current implementation.

Testing STIX Two Math Regular 2.13 b171 = it seems I cannot enable ssty=1 or ssty=2, it does nothing despite the documentation saying it's supported. I suspected it could be SILE's parsing of `+stty=value` in the C wrapper for Harfbuzz, but I cannot set it either in LibreOffice (though ss04 works). Dunno what to say.

Reply:
You know what? Why do I care what font support or not? Let's just implement the standard, and it's just nice Libertinus Math works well with it.
= rebased with a more standard-compliant approach, and now I am done with it I hope.